client/basic-authorship/src/basic_authorship.rs

```diff
@@ -91,7 +117,7 @@ impl<A, B, C> ProposerFactory<A, B, C> {
 	}
 }
 
-impl<B, Block, C, A> ProposerFactory<A, B, C>
+impl<B, Block, C, A, RP> ProposerFactory<A, B, C, RP>
```

Comment:
nitpick I guess this is named `RP` from `RecordProof` but probably `PR` would be better (`ProofRecording`).

Reply:
Ha! Before I renamed the trait over the lifetime of this pr :D